wp_api_integration_tests/tests/test_login_remote.rs

```diff
@@ -352,6 +353,30 @@ async fn login_spec_17_invalid_https_with_exception_works() {
     );
 }
 
+#[tokio::test]
+#[parallel]
+async fn login_spec_18_xmlrpc_disabled() {
+    // The xmlrpc endpoint does not return a valid HTTP response:
+    // $ curl https://xmlrpc-disabled.wpmt.co/xmlrpc.php
+    // curl: (92) HTTP/2 stream 1 was not closed cleanly: PROTOCOL_ERROR (err 1)
+    let result = xmlrpc_url("https://xmlrpc-disabled.wpmt.co").await;
+    assert!(result.is_err());
+    assert!(matches!(
+        result.unwrap_err(),
+        XMLRPCDiscoveryError::Disabled {
+            reason: XMLRPCDisabledReason::ByHost
+        }
+    ));
+}
```

Comment:
I want to add another test case for the `XMLRPCDisabledReason::ByPlugin` error. I haven't configured the test site yet, but will add one once the test site is ready.